vm/src/builtins/range.rs

```diff
@@ -1,3 +1,5 @@
+use std::cmp::max;
```

Comment:
The convention is mixed for now, but let's not make separated section of `use`. `rustfmt` will sort it.